gulpfile.js

```diff
@@ -179,6 +187,8 @@ gulp.task('build', [
   'build/zone.js',
   'build/zone.js.d.ts',
   'build/zone.min.js',
+  'build/zone-error.js',
```

Comment:
@mhevery , got it, I agree that to separate the error-rewrite to a stand alone module!
I will modify the document to decribe how to load zone.js and other stand alone modules(now we have `error-rewrite` and other non-standard api modules).  Just like the flags in add in `profiling.html`, we may have other stand alone modules (toString, geolocation...).

If we have too many small modules, should we provide some predefined module collections for users to load? 
such as `standard browser set`, `standard node set`, `standard electron set`, `basic set` and so on...